sdk/identity/azure-identity/azure/identity/_internal/shared_token_cache.py

```diff
@@ -159,7 +159,7 @@ def _get_cache_items_for_authority(
 
         cache = self._cae_cache if is_cae else self._cache
```

Comment:
Can we do it here:

cache: msal.TokenCache = cast(msal.TokenCache, self._cae_cache if is_cae else self._cache)

Reply:
Sure, moved the casts.